src/main/java/training/subsystems/elbow/ElbowNEO.java

```diff
@@ -6,7 +6,7 @@
 import edu.wpi.first.math.controller.ArmFeedforward;
 import edu.wpi.first.math.geometry.Rotation2d;
 
-public class ElbowNEO implements IElbow {
+public class ElbowNEO {
```

Comment:
why remove?

Reply:
In another commant Yoni wrote me that there is no need to get instance in ElbowNEO :)

Reply:
The Interface IElbow isnt an instance, Interfaces help us automatically write necessary commands when creating a new type of, in this case, an Elbow, u do need it.
Instances are the, well, Instance, of an object, in this case, neoElbow, the neoElbow could have an instance because there is only one, not multiple. But they arent really needed and you dont have to use them, there is no need.
all good :) cya tommorow

Reply:
Ok thanks for the explanation:) cu 2morrow